packages/circuits/circom/anon/poll.circom

```diff
@@ -3,7 +3,9 @@ pragma circom 2.0.0;
 // local imports
 include "../utils/hashers.circom";
 include "../utils/privToPubKey.circom";
-include "../trees/incrementalMerkleTree.circom";
+include "../trees/binary/MerkleTreeInclusionProof.circom";
```

Comment:
this template should only import "../trees/binary/BinaryMerkleRoot.circom"

Reply:
there might be mistakes in the other files too, please make sure you are importing only the files we need 🙏

Reply:
you can run the circuits tests to trigger the compilation and getting the stack trace telling you of any errors